### PR TITLE
Server side render

### PR DESCRIPTION
**Pull request by jonzee**

### CODE REVIEW

File: src/ts/Detail/FormVG.tsx [hunk 1]
```diff
@@ -143,6 +145,7 @@ export default class FormVG extends React.Component<any, any> {
           <div className="field">
             <FontIcon className="fa fa-angle-down" />
             <SelectField
+              id="select-equipment"
```
Comment (michalmikolajczyk) [inline src/ts/Detail/FormVG.tsx]: we should not use `id`s

File: src/ts/Detail/api.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import API from '../API';
 
 export function get(id) {
-  return fetch(`${API.offer}/${id}`, {
+  return window['fetch'](`${API.offer}/${id}`, {
```
Comment (michalmikolajczyk) [inline src/ts/Detail/api.ts]: is `window` always available?

**Comment by michalmikolajczyk** [comment]
please rebase, check the comments and let's merge this one in

**Comment by michalmikolajczyk** [comment]
or, rather, review the app on Heroku

File: src/ts/Detail/api.ts (hunk 2 above)
Comment (jonzee) [inline src/ts/Detail/api.ts]: I use api requests after user action or in `componentDidMount` for now, so it's not invoked on the server side.

**Comment by jonzee** [comment]
I deployed the app to heroku, it's ready to review

**Comment by michalmikolajczyk** [comment]
please check how the scraper sees our site: http://www.seo-browser.com/index.php

there doesn't seem to be any offers, and we need them.

https://viking-garage-frontend-s-pr-44.herokuapp.com/offer/2/honda-cb-125-1998-warsaw-poland this page actually has content from the server, but there is also a lot of trash like `keyboard-arrow-down`

**Pull request merged**